### PR TITLE
Fix travis push build: Travis + Docker + jsk_travis==0.4.34

### PR DESCRIPTION
**Pull request by wkentaro**
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1552
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1566
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1563
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1562
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1555
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1554
Close https://github.com/jsk-ros-pkg/jsk_common/pull/1552

### CODE REVIEW

**Comment by wkentaro** [comment]
screen shot 2017-11-01 at 18 28 20

I found that this approach is the only one solution at current. The test fail is caused by the issue reported/will be solved in https://github.com/jsk-ros-pkg/jsk_travis/pull/349.

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +17,9 @@ env:
     - ROS_PARALLEL_JOBS="-j8"
     - ROS_PARALLEL_TEST_JOBS="-j1"
     - CATKIN_PARALLEL_TEST_JOBS="-j1"
-    - USE_JENKINS=true
+    - USE_JENKINS=false
+    - USE_TRAVIS=true
```
Comment (furushchev) [inline .travis.yml]: @wkentaro What does it mean if `USE_TRAVIS` and `USE_DOCKER` are both true?
When I created this option, I assumed either one is true.

Comment (wkentaro) [reply .travis.yml]: I'm not sure, so I set `UES_JENKINS=false` and `USE_TRAVIS=true`.

**Comment by wkentaro** [comment]
Actually, I'm not sure how to fix below error only on precise:

Do you have any idea? @k-okada @furushchev 

```
No packages found matching iproute2.
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
jsk_tools: Cannot locate rosdep definition for [rosemacs]
jsk_data: No definition of [python-click] for OS version [precise]
libcmt: No definition of [libopencv-dev] for OS version [precise]
Continuing to install resolvable dependencies...
No packages found matching iproute2.
[1mexecuting command [sudo -H apt-get install -y -qq iproute2][0m
E: Unable to locate package iproute2
No packages found matching iproute2.
ERROR: the following rosdeps failed to install
  apt: command [sudo -H apt-get install -y -qq iproute2] failed
  apt: Failed to detect successful installation of [iproute2]
```

**Comment by k-okada** [comment]
please update to latest .travis.yaml, which does not require hydro test.

--
◉ Kei Okada


2017-11-02 23:42 GMT+09:00 Kentaro Wada <notifications@github.com>:

**Comment by furushchev** [comment]
Looks good to me.
@k-okada 
Please merge this pull request if it is ok for you?

**Pull request merged**